schema/container.go

```diff
@@ -7,13 +7,13 @@ import (
 )
 
 type ContainerRuntimeManifest struct {
-	ACVersion   types.SemVer            `json:"acVersion"`
-	ACKind      types.ACKind            `json:"acKind"`
-	UUID        types.UUID              `json:"uuid"`
-	Apps        AppList                 `json:"apps"`
-	Volumes     []types.Volume          `json:"volumes"`
-	Isolators   []types.Isolator        `json:"isolators"`
-	Annotations map[types.ACName]string `json:"annotations"`
+	ACVersion   types.SemVer       `json:"acVersion"`
+	ACKind      types.ACKind       `json:"acKind"`
+	UUID        types.UUID         `json:"uuid"`
+	Apps        AppList            `json:"apps"`
+	Volumes     []types.Volume     `json:"volumes"`
+	Isolators   []types.Isolator   `json:"isolators"`
+	Annotations []types.Annotation `json:"annotations"`
```

Comment:
Is this intentionally `[]types.Annotation` and not `types.Annotations` because you want to avoid the validation that the latter performs? 

If so, this is a subtlety that should be called out in the code and in the CRM section of the spec.

If not, use `types.Annotations` here, and update the CRM section of the spec to clarify that the same restrictions apply as to the `Annotations` field of the Image Manifest (e.g. "**annotations** arbitrary metadata appended to the app. Must be of the same type as the `annotations` field in the Image Manifest" or so)